src/socket-proxy/socket-proxyd.c

```diff
@@ -563,6 +564,7 @@ static void help(void) {
         printf("%1$s [HOST:PORT]\n"
                "%1$s [SOCKET]\n\n"
                "Bidirectionally proxy local sockets to another (possibly remote) socket.\n\n"
+               "  -c --connections-max   Set the maximum number of connections to be accepted\n"
```

Comment:
In the help text please also append "=" to the switch to indicate that this expects an argument.